### PR TITLE
Sponsor Done (commented out), Footer Fix, Nav Updates, All pages padding fix, Contact page frontend done

### PR DESCRIPTION
**Pull request by jayp822**
added sponsor us an our sponsor to nav bar and implemented the nav bar tab bold red when you are on that directory. Other fixes from un merged pull requests.

https://github.com/user-attachments/assets/8cc41403-4a35-4553-ac0c-f066086ae5c9

### CODE REVIEW

**Review by justinquinnb — COMMENTED** [review]
The sponsor pages were still in the design stage, they weren't ready for development. I'm so sorry I forgot to hide them on the Figma. Save your progress on them since the pages will be similar, but let's not publish those changes quite yet. All other changes look good though!

**Comment by jayp822** [comment]
Ok ill just comment it out for now

**Comment by justinquinnb** [comment]
Another comment I've just noticed--the text on the Our People page is incorrect. Looks like a simple copy-paste error from the Figma, as that's the text that's supposed to just be on the Our Projects page.

Intended text:
> DevDogs is proud to consist of over 180 UGA students from all tech disciplines and backgrounds. We firmly believe that different perspectives breed the best solutions--and we think our work shows! Take a look at some of the awesome people that have made DevDogs, DevDogs below.

**Comment by nivedhanatarajan7** [comment]
It seems like the display with different device widths are different. In the video you provided, it looks fine. However on some devices it looks like this 
image
image

**Comment by jayp822** [comment]
> In the video you provided, it looks fine. However on some devices it looks like this

fixed

File: src/app/about/page.js [hunk 1]
```diff
@@ -95,21 +107,33 @@ const AboutPage = () => {
...
+                    <br />
+                    <div className="flex flex-nowrap items-center justify-center ">
+                        <Link className="bg-NavBarColor rounded-full text-white mr-4 py-3 px-6 transition ease-in-out duration-200 hover:bg-black" href="/about/projects">
+                            <Button >All Projects</Button>
+                        </Link>
+                        <br />
+                        <Link className="bg-UGA rounded-full text-white ml-4 py-3 px-6 transition ease-in-out duration-200 hover:bg-black" href="/about/people">
+                            <Button >Our People</Button>
+                        </Link>
+                    </div> */}
```
Comment (ggliv) [inline src/app/about/page.js:122]:
```suggestion
```
Please don't do this. We have source control to track previous versions.

File: src/app/about/people/page.js [hunk 2]
```diff
@@ -1,31 +1,66 @@
...
 import ks from "../../images/officers/ks.jpg";
 import jb from "../../images/officers/jb.jpg";
 import am from "../../images/officers/am.jpg";
 import bm from "../../images/officers/bm.jpg";
 import kds from "../../images/officers/kds.jpg";
+import EmblaCarousel from "@/app/components/EmblaCarousel";
+import Caoursel1 from "@/app/images/carousel1.png";
+import Caoursel2 from "@/app/images/carousel2.jpg";
+import Caoursel3 from "@/app/images/carousel3.jpg";
+// import FlippableCard from "@/app/components/FlippableCard";
```
Comment (ggliv) [inline src/app/about/people/page.js:13]:
```suggestion
```

File: src/app/about/page.js (hunk 1 above)
Comment (ggliv) [inline src/app/about/page.js:122]:
```suggestion
```
Please don't do this. We have source control to keep track of previous versions.

File: src/app/components/FlippableCard.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+/* eslint-disable react/prop-types */
```
Comment (ggliv) [inline src/app/components/FlippableCard.js:1]: `PropTypes` are probably enabled by default for a reason. Please avoid lint overrides like this. If you think `PropTypes` should be disabled project-wide, open an issue and make your case there.

File: src/app/components/FlippableCard.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+// import React from "react";
+import Image from "next/image";
+// import { GitHubIcon } from "@/app/images/github.svg";
```
Comment (ggliv) [inline src/app/components/FlippableCard.js:4]:
```suggestion
import Image from "next/image";
```

**Review by ggliv — CHANGES_REQUESTED** [review]
Quick, incomplete glance over from a style perspective.

**Review by nivedhanatarajan7 — COMMENTED** [review]
The changes look good. Thank you!

**Pull request merged**